Source/WebCore/page/SecurityOriginData.cpp

```diff
@@ -68,7 +68,7 @@ SecurityOriginData SecurityOriginData::fromFrame(Frame* frame)
 
 Ref<SecurityOrigin> SecurityOriginData::securityOrigin() const
 {
-    return SecurityOrigin::create(protocol.isolatedCopy(), host.isolatedCopy(), port);
+    return SecurityOrigin::create(isolatedCopy());
```

Comment:
If `SecurityOrigin::create()` took a `SecurityOriginData&&`, we'd avoid some churn here.